backend/CircleForms/appsettings.Development.json

```diff
@@ -1,15 +1,23 @@
 {
   "Logging": {
-    "LogLevel": {
+    "MinimumLevel": {
       "Default": "Debug",
-      "Microsoft": "Information",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "Override": {
+        "Microsoft": "Information",
+        "Microsoft.Hosting.Lifetime": "Information",
+        "Hangfire": "Information"
+      }
     }
   },
   "ConnectionStrings": {
     "Database": "mongodb://mongodb:mongodb@mongodb",
     "Redis": "localhost:6379"
   },
+  "osuApi": {
```

Comment:
There's no need to have this in .Development, because we always need this in prod as well.

Reply:
It's in development so that you can have your dev keys here